slick/src/main/scala/slick/jdbc/PostgresProfile.scala

```diff
@@ -71,6 +71,7 @@ trait PostgresProfile extends JdbcProfile {
         case (VarCharPattern(str),"String") => Some(Some(str))
         case (IntPattern(v),"Int") => Some(Some(v.toInt))
         case (IntPattern(v),"Long") => Some(Some(v.toLong))
+        case (IntPattern(v),"Double") => Some(Some(v.toDouble))
```

Comment:
IntPattern sounds wrong. Shouldn't this allow for a decimal point? Also would be good to have a test for this. Not sure where the best place for the test is, but probably somewhere under https://github.com/slick/slick/tree/master/slick-testkit/src/codegen